### PR TITLE
Jakarta EE docs

### PR DESCRIPTION
**Pull request by lprimak**
Jakarta EE documentation

### CODE REVIEW

File: src/site/content/integration.adoc [hunk 1]
```diff
@@ -129,6 +129,8 @@ from https://twitter.com/lprimak[Lenny Primak].
 +
 Integrates Jakarta EE / Java EE applications with Shiro Security, specifically makes Shiro Annotations work with Java EE.
 
+NOTE: This module will be inegrated directly into Shiro with version 2.0 and above.
```
Comment (rzo1) [inline src/site/content/integration.adoc:132]: `inTegrated`

File: src/site/content/jakarta-ee.adoc [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+:jbake-type: page
+:jbake-status: published
+:jbake-tags: documentation, jakarta-ee, integrations, web
+:idprefix:
+:icons: font
+
+Apache Shiro Jakarta EE module makes it transparent to use Shiro features in Jakarta EE applications
+with minimal configuration. It makes annotations such as `@RequiresRoles` available in Jakarta EE (CDI, EJB, etc.) code.
+
+NOTE: Jakarta EE integration is available in Shiro 2.0 or later.
```
Comment (rzo1) [inline src/site/content/jakarta-ee.adoc:12]: Does it make sense to note which EE versions are compatible to the module? (as the whole thing was rebranded and we have the javax vs jakarta namespace thing)

**Review by fpapon — APPROVED** [review]
@lprimak Good job!

File: src/site/content/jakarta-ee.adoc (hunk 2 above)
Comment (bdemers) [reply src/site/content/jakarta-ee.adoc:12]: ➕ 1️⃣

File: src/site/content/integration.adoc (hunk 1 above)
Comment (lprimak) [reply src/site/content/integration.adoc:132]: fixed

File: src/site/content/jakarta-ee.adoc (hunk 2 above)
Comment (lprimak) [reply src/site/content/jakarta-ee.adoc:12]: done

**Pull request merged**